bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -692,42 +697,45 @@ def __init__(self, context, request):
-                        'getClientReference',
-                        'getClientSampleID',
-                        'ClientContact',
-                        'getSampleTypeTitle',
-                        'getSamplePointTitle',
-                        'getStorageLocation',
-                        'SamplingDeviation',
-                        # 'AdHoc',
-                        'getDateSampled',
-                        'getSampler',
-                        'getDatePreserved',
-                        'getPreserver',
-                        'getDateReceived',
-                        'getAnalysesNum',
-                        'getDateVerified',
-                        'state_title']},
+                         'getId',
+                         'getSample',
+                         'BatchID',
+                         # 'SubGroup',
+                         'Client',
+                         'Province',
+                         'District',
+                         'getProfilesTitle',
+                         'getTemplateTitle',
+                         'Creator',
+                         'Created',
+                         'getClientOrderNumber',
+                         'getClientReference',
+                         'getClientSampleID',
+                         'ClientContact',
+                         'getSampleTypeTitle',
+                         'getSamplePointTitle',
+                         'getStorageLocation',
+                         'SamplingDeviation',
+                         # 'AdHoc',
+                         'getDateSampled',
+                         'getSampler',
+                         'getDatePreserved',
+                         'getPreserver',
+                         'getDateReceived',
+                         'getAnalysesNum',
+                         'getDateVerified',
+                         'state_title']},
             {'id': 'unassigned',
              'title': "<img title='%s'\
-                       src='%s/++resource++bika.lims.images/unassigned.png'/>" % (
-                       t(_("Unassigned")), self.portal_url),
+                       src='%s/++resource++bika.lims.images/unassigned.png'/>"\
+                       % (t(_("Unassigned")), self.portal_url),
              'contentFilter': {'worksheetanalysis_review_state': 'unassigned',
-                               'review_state': ('sample_received', 'to_be_verified',
-                                                'attachment_due', 'verified',
-                                                'published'),
+                               'review_state': ('sample_registered',
+                                                'sample_prep',
+                                                'to_be_sampled',
+                                                'sample_due',
+                                                'sample_received',
+                                                'attachment_due',),
```

Comment:
`sample_registered` is a temporary state the Analysis Request reaches when is created, but the object is immediately transitioned to either `sampled` (if sampling workflow disabled) or `to_be_sampled` (if sampling workflow is enabled). In turn, if no preservation is required, an Analysis Request with `sampled` state is automatically transitioned to `sample_due`. In summary, querying for `sample_registered` should never return anything.

From my point of view, these are the states to be filtered by when `unassigned` filter is used: `sample_received`, `sample_due`, `sample_prep`, `to_be_preserved`, `to_be_sampled`, `attachment_due`

Reply:
My wrong. The assignment of analyses to Worksheets can only be done for those analyses that belong to Analysis Requests that are associated to a Sample Partitions that have been received. Therefore, there is no sense to add in this filter those states that are from the preservation or sampling workflow. Following this reasoning, I suggest this filter to consider the following states:

```python
{'worksheetanalysis_review_state': 'unassigned',
 'cancellation_state': 'active',
 'review_state': ('sample_received', 'attachment_due')
```

Related, but for analyses instead of Analysis Requests: [Analyses filter not kept after selecting one in dashboard #613](https://github.com/senaite/senaite.core/pull/613) . Note the comments:
- https://github.com/senaite/senaite.core/pull/613#discussion_r165438719
- https://github.com/senaite/senaite.core/pull/613#discussion_r165441820